manifests/operators/enable-cf-route-registrar.yml

```diff
@@ -58,6 +58,6 @@
   path: /releases/-
   value:
     name: routing
-    version: 0.209.0
-    url: https://bosh.io/d/github.com/cloudfoundry/routing-release?v=0.209.0
-    sha1: 4d5112c81d975b371f4b4513e112cb6054e5ed10
+    version: 0.317.0
+    url: https://d3r5lzz0qww9ge.cloudfront.net/routing-0.317.0.tgz
```

Comment:
This URL and sha does not match the values from here:

https://bosh.io/releases/github.com/cloudfoundry/routing-release?version=0.321.0